DQMOffline/Configuration/python/autoDQM.py

```diff
@@ -36,11 +40,11 @@
 
 	    'stripZeroBias': ['DQMOfflineTrackerStripMinBias',
 			      'PostDQMOffline',
-                              'DQMHarvestTrackerStrip+DQMCertTrackerStrip'],
+                              'DQMHarvestTrackerStrip'],
 
 	    'pixel': ['DQMOfflineTrackerPixel',
 		      'PostDQMOffline',
-                      'DQMHarvestTrackerPixel+DQMCertTrackerPixel'],
```

Comment:
This module was the cause of the crash, since it was originally removed for Phase1 wf. at the end of DQMOffline/Configuration/python/DQMOffline_Certification_cff.py
Until I realized that it took me a while. Now the configuration is much clearer IMHO so that this kind of mistakes should be less frequent now
I am very sorry. Thanks for the check!